src/j1cli.js

```diff
@@ -367,12 +366,17 @@ async function mutateAlertRules(j1Client, rules, update) {
   }
 }
 
+//j1Client,
+//Array.isArray(data) ? data : [data],
+//program.operation
 async function mutateQuestions(j1Client, questions, operation) {
   const created = [];
   const updated = [];
   const deleted = [];
   const skipped = [];
   const results = [];
+  var questionStrings = [];
```

Comment:
I don't see where this array is being used.

Reply:
Oh my fault I was trying to use that while returning the json file to the user but that didn't work out and I forgot to get rid of it.